intera_interface/src/intera_motion_interface/motion_trajectory.py

```diff
@@ -74,6 +74,10 @@ def stop_trajectory(self):
         self._client.stop_trajectory()
 
     def send_trajectory(self, wait_for_result=True, timeout=None):
```

Comment:
Total nitpick, and not really your code, however, we need to have a doc string for `wait_for_result`

Reply:
Sure. @MatthewPeterKelly what will be the best way to write it?
@param wait_for_result: If true then wait for trajectory to complete

Reply:
This entire class (and several others) appears to be poorly documented in 5.1 and well documented in 5.2. One option would be to copy the documentation from the 5.2 branch into this commit, the other is to just leave it and update the 5.2 documentation when this change is pushed. Copying from the 5.2 development branch:
````
def send_trajectory(self, wait_for_result=True):
        """
        Checks the trajectory message is complete.
        The message then will be packaged up with the MOTION_START
        command and sent to the motion controller.
        @param wait_for_result:
          - If true, the function will not until the trajectory is finished
          - If false, return True immediately after sending
        @return: True if the goal finished
        @rtype: bool
        """
````

Reply:
I have added documentation from 5.2 to 5.1 for this function